src/ProjectService.Data/Interfaces/IUserRepository.cs

```diff
@@ -19,7 +20,13 @@ public interface IUserRepository
         /// <param name="projectId">Project id from request.</param>
         void AddUsersToProject(IEnumerable<DbProjectUser> dbProjectUsers, Guid projectId);
 
-        IEnumerable<DbProjectUser> Find(Guid userId);
+        /// <summary>
+        /// Get user projects.
+        /// </summary>
+        /// <param name="userId">User Id from request</param>
+        IEnumerable<DbProjectUser> Get(Guid userId);
+        /// <param name="filter">Properties to filter query.</param>
+        IEnumerable<DbProjectUser> Get(GetDbProjectsUserFilter filter);
```

Comment:
Usually, we have one get with arg filter which includes id (but it is not a mistake I think). And get should return one entity, but here is the collection, I guess you have o rename it to find

Reply:
You are right